ui-components/src/headers/PageHeader.tsx

```diff
@@ -27,9 +39,24 @@ const PageHeader = (props: PageHeaderProps) => {
       <hgroup className="page-header__group">
         <h1 className="page-header__title">{props.title}</h1>
         {props.subtitle && <p className="page-header__lead"> {props.subtitle}</p>}
-      </hgroup>
+        {props.children}
 
-      {props.children}
+        {!!props?.tabNav?.length && (
+          <TabNav className="page-header__secondary-nav">
+            {props.tabNav?.map((tab) => (
```

Comment:
I'm wondering if we could use a different pattern here: passing items directly in as components rather than using a JSON intermediary. An example would be the `StatusAside` component, which accepts buttons via the `actions` prop. What do you think?

Reply:
Agree, it's a good idea, especially we have to get rid of any nextJS elements from ui-components (now it's still in the package but in the page_components dir).